addons/sys_bf888s/radio/fnc_getCurrentChannelInternal.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Returns the unique identity of the current channel.
+ *
+ * Arguments:
+ * 0: Radio ID <STRING> (Unused)
```

Comment:
Is it really unused ? It's reffed in the code below, so I suspect it's an accidental copypaste.

Reply:
```suggestion
 * 0: Radio ID <STRING>
```